Error: Cannot find module 'bug-killer'

# This Issue
Whenever you clone the project and run `npm start` you get **Error: Cannot find module 'bug-killer'**.

![image](https://user-images.githubusercontent.com/12969835/32140256-f2723b9c-bc27-11e7-8570-6906361d3f5b.png)

# The Solution
To fix the issue run `npm install bug-killer`. The package.json for the project should be updated.

--- a/package.json
+++ b/package.json
@@ -9,6 +9,7 @@
   ],
   "main": "index.js",
   "dependencies": {
+    "bug-killer": "^4.4.4",
     "lien": "^1.0.2",
     "opn": "^4.0.1",
     "r-json": "^1.2.2",

